packages/now-go/index.ts

```diff
@@ -136,8 +142,12 @@ Learn more: https://github.com/golang/go/wiki/Modules
     if (fileName in downloadedFiles) {
       goModAbsPathDir = dirname(downloadedFiles[fileName].fsPath);
       debug(`Found ${fileName} file in "${goModAbsPathDir}"`);
+    } else if ('api/go.mod' in downloadedFiles) {
```

Comment:
What if I have `api/foo/go.mod`? I feel like we should be using `scanParentDirs()` or whatever that function is called to iterate up and find the `go.mod` file.

Reply:
@TooTallNate We don't allow that behavior and I don't think Go does either. This was really just a feature for users to separate backend and frontend (backend being everything in the api directory).